Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsoTrk_Output_cff.py

```diff
@@ -11,14 +11,17 @@
     outputCommands = cms.untracked.vstring( 
         'keep *_IsoProd_*_*',
         'keep *_TkAlIsoProd_*_*',
+        'keep *_hbhereco_*_*',
```

Comment:
@bsunanda right, still it would be good to have at least qualitatively a measure of it. how urgent is this? 2016 pp data-taking is over.

@ghellwig, unfortunately none of those: 
https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Configuration/AlCa/python/autoAlca.py#L3 
on data this is run on the Commissioning PD (we'll need to see if there is any RAW around)